src/domain/accounts/domain/src/entities/predefined_accounts_config.rs

```diff
@@ -98,6 +99,8 @@ pub struct AccountConfig {
     #[serde(default)]
     pub is_admin: bool,
     #[serde(default)]
+    pub can_provision_accounts: bool,
```

Comment:
Not necessarily now, but I think it would make more sense for us to model these not as `is_x` flags but rather a list of `properties: [admin, can_provision_accounts]` and later relations. Otherwise we'll:
- get tired extending this type with all possible properties
- make system rigid - we want to allow for all kinds of custom ReBAC properties here